instructions/index.js

```diff
@@ -0,0 +1,15 @@
+let today = new Date();
+let thisYear = today.getFullYear();
+let footer = document.querySelector("footer");
+let copyright = document.createElement("p");
+copyright.innerText = "Vanessa Rodriguez " + thisYear;
+footer.appendChild(copyright);
+
+const skills = ["Patient Assessment", "HTML", "CSS", "JavaScript", "Medical Terminology"];
+let skillsSection = document.querySelector("skills");
```

Comment:
This selector won't actually find anything. When you use `document.querySelector`, you have to provide some special characters to tell the function what type of thing you're looking for (a class, and ID, a type of HTML element, etc.). The default is looking for HTML elements, so this selector would be looking for a `<skills>` element, which of course does not exist. To select an element by ID, you have to prefix it with `#`: `document.querySelector('#skills')`

Reply:
Some docs for reference if you want them: https://developer.mozilla.org/en-US/docs/Web/API/Document_object_model/Locating_DOM_elements_using_selectors#examples